src/theme.ts

```diff
@@ -0,0 +1,156 @@
+import { createMuiTheme } from "@material-ui/core";
+
+const defaultFonts = [
+  "Open Sans",
+  "Arial",
+  "Helvetica Neue",
+  "Helvetica",
+  "sans-serif",
+].join(",");
+
+const headingFonts = ["Poppins", "sans-serif"].join(",");
+
+export const white = "#FFFFFF";
+export const black = "#000000";
+export const grey = "#868686";
+export const lavenderBlush = "#FFF3F7";
+export const azalea = "#F7C0CB";
+export const mauvelous = "#EF7D95";
+export const jordyBlue = "#88B6EC";
+export const springRain = "#ADCDB8";
+export const rajah = "#F6B45C";
+export const jaffa = "#F07A48";
+
+const textColor = black;
+const contrastText = white;
+const primaryColor = azalea;
+const secondaryColor = jordyBlue;
+
+const theme = createMuiTheme({
+  palette: {
+    background: {
+      default: white,
+    },
+    primary: {
+      main: primaryColor,
+      contrastText,
+    },
+    secondary: {
+      main: secondaryColor,
+      contrastText,
+    },
+    success: {
+      main: springRain,
+      contrastText,
+    },
+    error: {
+      main: jaffa,
+      contrastText,
+    },
+    info: {
+      main: grey,
+      dark: black,
+    },
+    text: {
+      primary: textColor,
+    },
+  },
+  typography: {
+    /* 
+      `htmlFontSize` is the font size for the <html> element, browser default is 16px, (62.5% of 16px = 10px)  
+    */
+    htmlFontSize: 10,
+    fontFamily: defaultFonts,
+    h1: {
+      fontFamily: headingFonts,
+      color: textColor,
+      fontWeight: 700,
+      fontSize: "36px",
+      // h1 = Title 1
+    },
+    h2: {
+      fontFamily: headingFonts,
+      color: textColor,
+      fontWeight: 700,
+      fontSize: "30px",
+      // h2 = Title 2
+    },
+    h3: {
+      fontFamily: headingFonts,
+      color: textColor,
+      fontWeight: 700,
+      fontSize: "25px",
```

Comment:
Should: change to 24px (although it differs from design system)

Reply:
```suggestion
      fontSize: "24px",
```